specifications/FiniteMonotonic/SCCRDT.tla

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Would be nice to add a longer comment explaining what this spec is doing

Reply:
Also to be removed